cmake/Modules/FindRTICivetweb.cmake

```diff
@@ -0,0 +1,139 @@
+# (c) 2023 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the software solely for use with RTI Connext DDS.  Licensee may
+# redistribute copies of the software provided that all such copies are
+# subject to this license. The software is provided "as is", with no warranty
+# of any type, including any warranty for fitness for any purpose. RTI is
+# under no obligation to maintain or support the software.  RTI shall not be
+# liable for any incidental or consequential damages arising out of the use or
+# inability to use the software.
+
+#[[.rst:
+.. _find_rticivetweb:
+
+FindRTIOCivetweb
+--------------
+
+Find the Civetweb libraries.
+If no RTICivetweb_ROOT is provided it will try to find the libraries in the
```

Comment:
Do you mean, "If no RTICivetweb_ROOT is provided, FindRTIOCivetweb will try to find the RTIOCivetweb libraries in the Connext installation"?

Reply:
What about:

> If no RTICivetweb_ROOT is provided, FindRTICivetweb will try to find the Civetweb libraries in the Connext installation.

Reply:
sounds good!